test/debug/session_test.rb

```diff
@@ -18,15 +18,15 @@ def program
 
     def test_session_starts_manually
       debug_code(program, boot_options: "") do
-        assert_line_num(5)
+        assert_line_num(4)
```

Comment:
This change actually makes `DEBUGGER__.console` stop more accurately. It used to stop at the next line but it now stops at the same line.